converter/src/main/java/gov/cms/qpp/conversion/model/error/Detail.java

```diff
@@ -31,42 +39,51 @@ public Detail() {
 	 * Copy constructor
 	 */
 	public Detail(Detail detail) {
-		this(detail.getMessage(), detail.getPath(), detail.getValue(), detail.getType());
+		setErrorCode(detail.getErrorCode());
+		setMessage(detail.getMessage());
+		setPath(detail.getPath());
+		setValue(detail.getValue());
+		setType(detail.getType());
 	}
 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1213)